application/src/sass/components/_buttons.scss

```diff
@@ -113,17 +113,25 @@ $eff-quiet-button-hover-colour: mix($black, $eff-quiet-button-colour, 15%);
   cursor: pointer;
 }
 
-.eff-button--link:active {
+.eff-button-link:active {
   color: govuk-colour("blue");
 }
 
-.eff-button--link:focus,
-.eff-button--link:focus:hover {
+.eff-button-link:focus,
+.eff-button-link:focus:hover {
   background-color: $govuk-focus-colour;
   color: $govuk-focus-text-colour;
 }
 
-.eff-button--link:hover {
+.eff-button-link:hover {
   color: $govuk-link-hover-colour;
   background-color: transparent;
 }
+
+.eff-button-link--destructive {
```

Comment:
Let's use "warning" to be consistent with https://design-system.service.gov.uk/components/button/#warning-buttons?

Also I think this should probably be either `.eff-button--warning-link` (as an alternative to `.eff-button--link`) or `.eff-button--warning` used alongside `.eff-button--link`, as I'm not sure BEM lets you specify two modifiers in the same class?

Reply:
I actually changed it to `eff-button-link` as the name of the component, because we probably want to maintain that as its own component, which can have variants - e.g. looking like normal links, or warning links.

So we'll have `eff-button-link`, and `eff-button-link--warning`. This avoids the double-modifier thing you're concerned about.

Reply:
Have updated it to be `warning`